app/src/main/res/values-hr-rHR/strings.xml

```diff
@@ -1,114 +1,114 @@
-    <string name="accent_color">Accent Color</string>
-    <string name="accent_blue">Blue</string>
-    <string name="accent_green">Green</string>
-    <string name="accent_purple">Purple</string>
-    <string name="accent_red">Red</string>
-    <string name="accent_yellow">Yellow</string>
-    <string name="category_appearance">Appearance</string>
-    <string name="category_behaviour">Behavior</string>
-    <string name="clear_files">Clear downloaded files</string>
-    <string name="cleared_files">Successfully cleared files</string>
-    <string name="firebase_title">Firebase Analytics</string>
-    <string name="firebase_summary">This lets us collect information about app performance and crash logs</string>
-    <string name="language_title">Language</string>
-    <string name="link_title">Use Chrome Custom Tabs</string>
-    <string name="link_custom_tabs">Links will open in Chrome Custom Tabs</string>
-    <string name="system_default">System Default</string>
-    <string name="theme">Theme</string>
-    <string name="theme_dark">Dark Theme</string>
-    <string name="theme_light">Light Theme</string>
-    <string name="update_url">Update Channel URL</string>
-    <string name="push_notifications">%1$s Push Notifications</string>
-    <string name="push_notifications_summary">Receive push notifications when an update for %1$s is released</string>
-    <string name="update_center">Manager Update Center</string>
-    <string name="update_not_found">No new updates</string>
-    <string name="variant">Variant</string>
+    <string name="accent_color">Boja naglašavanja</string>
+    <string name="accent_blue">Plava</string>
+    <string name="accent_green">Zelena</string>
+    <string name="accent_purple">Ljubičasta</string>
+    <string name="accent_red">Crvena</string>
+    <string name="accent_yellow">Žuta</string>
+    <string name="category_appearance">Izgled</string>
+    <string name="category_behaviour">Ponašanje</string>
+    <string name="clear_files">Ukloni preuzete datoteke</string>
+    <string name="cleared_files">Datoteke su uspješno uklonjene</string>
+    <string name="firebase_title">Firebase analitika</string>
+    <string name="firebase_summary">Ovo nam omogućuje prikupljanje informacija o performansama aplikacije i zapise rušenja</string>
+    <string name="language_title">Jezik</string>
+    <string name="link_title">Koristi prilagođene kartice Chroma</string>
+    <string name="link_custom_tabs">Poveznice će se otvarati u prilagođenim karticama Chroma</string>
+    <string name="system_default">Zadan sustavom</string>
+    <string name="theme">Tema</string>
+    <string name="theme_dark">Tamna tema</string>
+    <string name="theme_light">Svjetla tema</string>
+    <string name="update_url">Ažuriraj URL kanala</string>
+    <string name="push_notifications">%1$s skočnih obavijesti</string>
+    <string name="push_notifications_summary">Primite skočnu obavijest kada je ažuriranje za %1$s dostupno</string>
```

Comment:
I would suggest using "Primite skočnu obavijest kada ažuriranje za %1$s postane dostupno"

Translated it would be: "Receive push notifications when an update for %1$s becomes available"